### ISSUE TITLE
Actually allow other people to use this plugin

### ISSUE THREAD

**Issue by Purrrpley** [AutismFather/Hug#2]
Currently, in `Hug.java`, there's these 4 lines:
```java
/*
 * This plugin was created by AutismFather for the Autcraft and Autcraft Obsidian servers.
 * No other servers are permitted to use this plugin in part or in it's entirety.
 */
```
This is likely left over from before its source code was published here, and from my understanding means that no one else is actually allowed to use this plugin, which doesn't make much sense and so is likely unintended. That comment should maybe be removed, and maybe replaced with a proper license for this repository.

A proper license for this repository, like one from [choosealicense.com](https://choosealicense.com/), would also allow other people to legally fork this repository so they can add new features and stuff to it. It would also make this plugin open source instead of just source available (I think, none of this is legal advice but is correct to my knowledge).

**Comment by ghost** [comment]
I personally prefer open-source software to have an open-source license/FOSS license, but in the end, go for whatever it should be. 😄 (And yes, @Purrrpley, your understanding is correct.)

**Comment by Purrrpley** [comment]
This got fixed like 8 months ago on 2022-09-07 in 210377ac9e8be3fdd84f04695e895580cfa424b2 and I missed it until just now, oops >.< this issue has been fixed! No open source license though, still [no license](https://choosealicense.com/no-permission/), but people can actually use this plugin now when getting it from GitHub and not the Spigot page (I think that's how it works and the only change this made)
